amp-camp/src/html/pages/product-listing.html

```diff
@@ -5,7 +5,6 @@
     %%include('../partials/head.html', {
         "pageType": "product-listing"
     })
-    <link rel="preload" href="/api/categories?categoryId=<%productsGender%>-<%productsCategory%>&sort=high-low" as="fetch">
```

Comment:
Can you explain why this doesn't work anymore and should be removed?

Reply:
This is for the amp-list isn't it? The amp-runtime adds custom headers and query parameters to the URL which result in the preload not working:

`https://camp.samples.amp.dev/api/categories?categoryId=women-shirts&sort=high-low&__amp_source_origin=https%3A%2F%2Fcamp.samples.amp.dev`

Reply:
Good point, I noticed that! I was wondering if the extra params could be hardcoded in the preload or are they generated on the fly? Is there a workaround?

Thanks.

Reply:
I'm not aware of any workarounds.

Reply:
Dang.

Antoine, I was so excited by this idea. Now I suppose we know why our friends at Womp weren't already doing this....